src/main/java/uk/gov/companieshouse/company/profile/util/LinkRequest.java

```diff
@@ -6,9 +6,9 @@ public class LinkRequest {
     public static final String EXEMPTIONS_DELTA_TYPE = "exemption_delta";
     public static final String OFFICERS_LINK_TYPE = "officers";
     public static final String OFFICERS_DELTA_TYPE = "officer_delta";
-    public static final String PSCSTATEMENTS_LINK_TYPE =
+    public static final String PSC_STATEMENTS_LINK_TYPE =
             "persons_with_significant_control_statements";
-    public static final String PSCSTATEMENTS_DELTA_TYPE = "PscStatement_delta";
+    public static final String PSC_STATEMENTS_DELTA_TYPE = "PscStatement_delta";
```

Comment:
Could the value for this be changed as well? I think it should probably be psc_statement_delta as opposed to PscStatement_delta